pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -860,3 +898,60 @@ func (k *Bootstrapper) elevateKubeSystemPrivileges(cfg config.ClusterConfig) err
 	}
 	return nil
 }
+
+
+// adviseNodePressure will advise the user what to do with difference pressure errors based on their environment
+func adviseNodePressure(err error, name string, drv string) {
+	if diskErr, ok := err.(*kverify.ErrDiskPressure); ok {
+		out.ErrLn("")
+		glog.Warning(diskErr)
+		out.WarningT("The node {{.name}} has ran out of disk space.", out.V{"name": name})
+		// generic advice for all drivers
+		out.T(out.Tip, "Please free up disk or prune images.")
```

Comment:
The generic case should be a fall-back, not a default. This message seems to direct users to freeing up disk space or pruning their host, rather than within the container or VM.

Reply:
this tells ppl they need to free UP disk
and next lines it tells them how to do it, based on their platform.
and it makes the lines shorter and nicer icons. If u dont mind I will keep this

Reply:
additionaly leave this for a follow up PR for UI refrewsh